src/rest/main/main.go

```diff
@@ -55,7 +55,7 @@ func init() {
 	flag.StringVar(&keyFile, "key", "", "Server private key file path")
 	flag.StringVar(&caFile, "cacert", "", "CA certificate for client certificate validation")
 	flag.StringVar(&cliCAFile, "clicacert", "", "CA certificate for CLI client validation")
-	flag.Var(clientAuth, "client_auth", "Client auth mode(s) - <cert,password,jwt,none> default: password,jwt")
+	flag.Var(clientAuth, "client_auth", "Client auth mode(s) - <none,cert,jwt,password|user(depricated)> default: password,jwt")
```

Comment:
"depricated" should be spelled "deprecated"

Reply:
whoops, fixed.

Reply:
fixed